lib/trino-parquet/src/main/java/io/trino/parquet/predicate/TupleDomainParquetPredicate.java

```diff
@@ -368,11 +470,21 @@ private static ParquetCorruptionException corruptionException(String column, Par
         return new ParquetCorruptionException(format("Corrupted statistics for column \"%s\" in Parquet file \"%s\": [%s]", column, id, statistics));
     }
 
+    private static ParquetCorruptionException corruptionException(String column, ParquetDataSourceId id, ColumnIndex columnIndex)
+    {
+        return new ParquetCorruptionException(format("Corrupted statistics for column \"%s\" in Parquet file \"%s\": [%s]", column, id, columnIndex));
```

Comment:
I'd rephrase this as `Corrupted statistics for column "%s" in Parquet file "%s". Corrupted column index: %s` to make the last part as clear as possible.